.github/workflows/update-flake-lock.yml

```
@@ -0,0 +1,66 @@
+        run: |
+          NEW_REV=$(jq -r '.nodes."hydenix-nixpkgs".locked.rev' flake.lock)
+          if [ -n "$NEW_REV" ] && [ "$NEW_REV" != "null" ]; then
+            sed -i 's|hydenix-nixpkgs.url = "github:nixos/nixpkgs/[^"]*"|hydenix-nixpkgs.url = "github:nixos/nixpkgs/'$NEW_REV'"|' flake.nix
+          else
+            echo "Failed to extract revision from flake.lock"
+            exit 1
+          fi
+      - name: Verify flake.nix
+        run: |
+          grep -q "hydenix-nixpkgs.url" flake.nix || (echo "Failed to update flake.nix" && exit 1)
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)